.github/workflows/percy_snapshots.yml

```diff
@@ -55,3 +55,4 @@ jobs:
           PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
           PERCY_PARALLEL_NONCE: ${{ inputs.PERCY_PARALLEL_NONCE }}
           PERCY_PARALLEL_TOTAL: ${{ inputs.PERCY_PARALLEL_TOTAL }}
+          REACT_APP_MAPBOX_API_KEY: ${{ secrets.REACT_APP_MAPBOX_API_KEY }}
```

Comment:
I think you'll also need to add `REACT_APP_MAPBOX_API_KEY` as an input secret above. you can make sure by testing your changes on this `dev` branch in your `search-ui-react` PR by switching out the resusable workflows reference to refer to `@<this-branch>` instead of `@v1`